protocol/messenger_communities.go

```diff
@@ -6148,3 +6148,11 @@ func (m *Messenger) CreateResponseWithACNotification(communityID string, acType
 
 	return response, nil
 }
+
+func (m *Messenger) SendCommunityMessage(community *communities.Community, rawMessage common.RawMessage) ([]byte, error) {
+	if community.ControlNode() != nil && !community.PublicKey().Equal(community.ControlNode()) {
```

Comment:
```suggestion
	if !community.PublicKey().Equal(community.ControlNode()) {
```

ControlNode() can't be nil, can it? :thinking:

Reply:
just added it for backward compatibility, I'm not sure that on the previous version of status it can't be nil

Reply:
I don't understand the concern :thinking: you are not pushing this commit in the past, are you?

https://github.com/status-im/status-go/blob/c62957f621ff8169f5f4f0650ac4ccf71a1cafa9/protocol/communities/community.go#L1352-L1357